<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblControlVentas">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CONTROL DE REGISTRO DE VENTAS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblFecha">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>75</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fecha actual</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtFecha">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>75</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2023</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>541</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Área de venta</string>
    </property>
    <widget class="QLabel" name="lblSeleccionarProducto">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>35</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar producto</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSeleccionarPago">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>95</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar tipo de pago</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboProducto">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>65</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Seleccionar producto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Colección Escolar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Colección PreUniversitaria</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Colección Profesional</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cboPago">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>125</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Seleccionar Tipo de Pago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Contado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tarjeta</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblCantidad">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>35</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPrecioProducto">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>35</y>
       <width>130</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Precio del Producto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCantidad">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>65</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPrecio">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tblDatos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>PRDUCTO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CANTIDAD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PRECIO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TIPO PAGO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DESCUENTO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RECARGO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PRECIO FINAL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>FECHA</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>salir1.png</normaloff>salir1.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>REGISTRAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>registrar.png</normaloff>registrar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNuevo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>NUEVO</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>nuevo.png</normaloff>nuevo.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
